addons/l10n_in_edi_ewaybill/models/account_move.py

```diff
@@ -124,3 +124,9 @@ def l10n_in_edi_ewaybill_send(self):
                 })
         self.env['account.edi.document'].create(edi_document_vals_list)
         self.env.ref('account_edi.ir_cron_edi_network')._trigger()
+
+    def _can_force_cancel(self):
+        # OVERRIDE
+        """ Mark this invoice cencel(forcefully) without waiting for the EDI document to be canceled. """
```

Comment:
```suggestion
```
nitpicking: personally, I think it's not needed, but feel free to keep it (making it consistent with the inherited method) if you prefer